MatrixSDK/Crypto/Data/Store/MXRealmCryptoStore/MXRealmCryptoStore.m

```diff
@@ -416,10 +416,13 @@ + (void)_deleteStoreWithCredentials:(MXCredentials*)credentials readOnly:(BOOL)r
                 [realm transactionWithBlock:^{
                     [realm deleteAllObjects];
                 }];
+                
+                // We must keep the last version
+                [self setVersion:MXCryptoVersionLast];
```

Comment:
This seems useless to me. We don't have a stored `version` property on this class.

You probably wanted to use `cryptoVersion` but that won't help neither, because we removed the account object above. Maybe we need a metadata object to store the version separately.

Reply:
Argh. Thanks. Autocompletion led me to a wrong thing.